schemas/string_interpolation/definitions.json

```diff
@@ -76,6 +76,12 @@
             {
               "$ref": "https://eq.ons.gov.uk/string_interpolation/transforms/format_number.json#/format_number"
             },
+            {
+              "$ref": "https://eq.ons.gov.uk/string_interpolation/transforms/format_percentage.json#/format_percentage"
+            },
+            {
+              "$ref": "https://eq.ons.gov.uk/string_interpolation/transforms/format_unit.json#/format_unit"
+            },
```

Comment:
We should add these to a valid schema also.

Reply:
Added a valid schema from runner, with format unit and percentage transforms.